akka-docs/rst/scala/http/routing-dsl/directives/basic-directives/index.rst

```diff
@@ -20,6 +20,7 @@ a single value or a tuple of values.
   * :ref:`-extractDataBytes-`
   * :ref:`-extractExecutionContext-`
   * :ref:`-extractMaterializer-`
+  * :ref:`-extractStrictEntity-`
```

Comment:
need to provide these in `akka-docs/rst/java/http/routing-dsl/directives/basic-directives/index.rst` too